tox.ini

```diff
@@ -51,8 +51,9 @@ description = [run locally after tests]: combine coverage data and create report
 skip_install = true
 deps =
     covdefaults>=2.2.2
-    coverage[toml]>=7.0.1
     diff_cover>=7.3
+    git+https://github.com/nedbat/coveragepy@nedbat/fix-virtualenv#egg=coverage==0.0
+    tomli
 extras =
```

Comment:
Why do we need this, isn't the `toml` extra already covering this?

Reply:
That was just for my debugging, I wasn't sure of the extra syntax with a git+ url, so I just stuck it in there like this.

Reply:
You can revert these lines. Coverage.py 7.0.2 is now out, with the fix.